.teamcity/_self/projects/WebApp.kt

```diff
@@ -404,6 +404,9 @@ object RunAllUnitTests : BuildType({
 				unset NODE_ENV
 				unset CALYPSO_ENV
 
+				# Decrypt calypso-e2e secrets
+				E2E_SECRETS_KEY="%E2E_SECRETS_ENCRYPTION_KEY_CURRENT%" yarn workspace @automattic/calypso-e2e decrypt-secrets
```

Comment:
This PR requires interaction with the `calypso-e2e` secrets manager.

Reply:
> This also means people will have to encrypt the E2E secrets file on their machine if they want to run unit tests, and that feels like it could be pretty disruptive.

I hadn't considered that possibility but I think you are correct that it would require developers to _decrypt_ the secrets, but only if they're running the entire suit of tests under `packages/` - which is an unlikely proposition, but one that's not entirely impossible I admit.

Reason I say that this is unlikely is because I imagine that developers would typically like to use the `watch` variant so that each change don't trigger a 10-minute test run.

In either case, as you suggested in a subsequent review to eliminate the integration part of these tests, I think it would make sense to mock the `SecretsManager` instead.